Userland/Libraries/LibWeb/XHR/ProgressEvent.idl

```diff
@@ -12,6 +12,6 @@ interface ProgressEvent : Event {
 
 dictionary ProgressEventInit : EventInit {
     boolean lengthComputable = false;
-    unsigned long loaded = 0;
-    unsigned long total = 0;
+    unsigned long long loaded = 0;
+    unsigned long long total = 0;
```

Comment:
Can we actually instantiate `ProgressEventInit` with huge 64-bit int values from JavaScript though? :yakthonk:

Reply:
I believe so, unless I am mistaken: https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibWeb/WebIDL/AbstractOperations.cpp#L344